xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/AllAttachments.xml

```diff
@@ -38,28 +38,35 @@
   <hidden>true</hidden>
   <content>{{velocity}}
 #set ($discard = $xwiki.ssfx.use('js/xwiki/viewers/attachments.css', true))
-#set ($discard = $xwiki.jsx.use('XWiki.AllAttachments'))
-#set ($columns = ['mimeType', 'filename', 'doc.fullName', 'filesize', 'date', 'author'])
-#set ($columnsProperties = {
-  'mimeType': {'type': 'list', 'class': 'XWiki.AllAttachments'},
-  'filename': {'type': 'text'},
-  'doc.fullName': {'type': 'text'},
-  'filesize': {'type': 'list', 'class': 'XWiki.AllAttachments'},
-  'date': {'type': 'date'},
-  'author': {'type': 'text'}
-})
-#set ($options = {
+#set ($sourceParameters = $escapetool.url({
+  'className': 'XWiki.AllAttachments',
   'resultPage': 'XWiki.AllAttachmentsResults',
-  'callback': 'XWiki.index.displayAttachmentEntry',
-  'translationPrefix': 'platform.index.attachments.',
-  'selectedColumn': 'filename'
+  'translationPrefix': 'platform.index.attachments.'
+}))
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+      { 'id': 'mimeType', 'displayer': 'html'},
+      { 'id': 'filename', 'displayer': { 'id': 'link', 'propertyHref': 'fileurl' } },
+      { 'id': 'doc.fullName', 'displayer': 'html' },
+      { 'id': 'filesize', 'displayer': 'html' },
+      { 'id': 'date', 'filter': 'date'},
+      { 'id': 'author', 'displayer': 'html' }
+    ],
+    'actions': [
+      {'id': 'view', 'allowProperty': 'acclev'}
+    ]
+  }
 })
-#livetable('allattachments' $columns $columnsProperties $options)
-{{/velocity}}
 
-(% id="inaccessibleDocsMessage" class="hidden" %)(((
-  {{info}}{{translation key="rightsmanager.documentrequireviewrights" /}}{{/info}}
-)))</content>
```

Comment:
Is it expected that this piece of code, along with the corresponding javascript, are moved?

Reply:
Yes, Live Data has this integrated. There is a screenshot demonstrating this in this pull request.

Reply:
indeed :) All good then! Thanks